vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -79,6 +79,13 @@
     // the selected participants list
     public static final String EXTRA_OUT_SELECTED_PARTICIPANT_ITEMS = "VectorInviteMembersActivity.EXTRA_OUT_SELECTED_PARTICIPANT_ITEMS";
 
+    // This enum is used to filter the display of the contacts
+    public enum ContactsFilter { ALL, TCHAP_ONLY, NO_TCHAP_ONLY }
+    private ContactsFilter contactsFilter = null;
+
+    // This enum is used to select a mode for room creation (DIRECT_CHAT or DISCUSSION)
```

Comment:
Do not mention the values in comment; Most of the time the comment is not updated when the values change ;)